roles/execution/templates/testnet/normal_env_var

```diff
@@ -0,0 +1,10 @@
+GETH_DATADIR={{ execution_data_dir }}
+GETH_SYNCMODE={{ execution_syncmode }}
+GETH_METRICS=true
+GETH_HTTP=true
+GETH_HTTP_PORT={{ vars[project][type][client].ports.http_port }}
+GETH_PORT={{ vars[project][type][client].ports.p2p_port }}
+GETH_DISCOVERY_PORT={{ vars[project][type][client].ports.network_discovery_port }}
+GETH_METRICS_PORT={{ vars[project][type][client].ports.metrics_port }}
+GETH_AUTHRPC_PORT={{ vars[project][type][client].ports.auth_rpc_port }}
```

Comment:
or all these variables you shouldnt use vars[project][type][client], use it in the files where should things can change, here you know already that project is storyprootocl and client is execution

Reply:
also after we port prefix, this should change